<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLiveCapture</class>
 <widget class="QDialog" name="DlgLiveCapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Live Capture in progress</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lHeader">
     <property name="text">
      <string>VisualMassifDiff is now capturing live data from a valgrind instance</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lNumSnapshots">
       <property name="text">
        <string>Snapshots captured so far</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lNumSnapshotsValue">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lSecondsTillNext">
       <property name="text">
        <string>Seconds till next capture</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="pbSecondsTillNext">
       <property name="value">
        <number>24</number>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txtLog">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bClose">
       <property name="text">
        <string>Stop capture</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bClose</sender>
   <signal>clicked()</signal>
   <receiver>DlgLiveCapture</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>867</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
